automation/video.py

```
@@ -20,6 +41,51 @@ def __init__(self, page: Page, auth_manager):
         self.page = page
         self.auth_manager = auth_manager
 
+    async def ensure_video_playing(self, video_selector: str = "video") -> dict:
+        """
+        确保视频正在播放，如果暂停则自动恢复，并返回视频状态
```

Ruff (0.14.10)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)